adal/authority.py

```diff
@@ -84,11 +90,14 @@ def _parse_authority(self):
             self._tenant = path_parts[1]
         except IndexError:
             raise ValueError("Could not determine tenant.")
-
+ 

```

Comment:
Nitpicking: would you mind remove the unnecessary space(s) introduced in this PR? See the explanation in another comment below.

Reply:
Removed unnecessary space